backend/zeno_backend/processing/chart.py

```diff
@@ -17,11 +17,74 @@
 from zeno_backend.classes.filter import FilterPredicateGroup, Join
 from zeno_backend.classes.metric import Metric
 from zeno_backend.classes.slice import Slice
-from zeno_backend.database.select import metrics, slices
+from zeno_backend.database.select import metrics, models, slices
 from zeno_backend.processing.filtering import table_filter
 from zeno_backend.processing.metrics.map import metric_map
 
 
+def get_selected_slices(chart_slices: list[int], project: str) -> list[Slice]:
+    """Get the slices of the chart from a list of slice ids.
+
+    Args:
+        chart_slices (list[int]): slice ids of the chart.
+        project (str): project the user is currently working with.
+
+    Returns:
+        list[Slice]: list of slices of the chart.
+    """
+    if -2 in chart_slices:  # -2 signals that all slices should be selected
```

Comment:
Should this include `all_instances`? I can go either way.

Reply:
Don't know either.... Often destroys the scales that's why I did not include it.

Reply:
Yeah I'm alright not including it.